### ISSUE TITLE
How did you split the training/validation sets?

### ISSUE THREAD

**Issue by xianzhidu** [JiaRenChang/PSMNet#31]
Can you share which images are used for validation? Thanks

**Comment by JiaRenChang** [comment]
Hi, @xianzhidu 
The name numbers of image were:
[1,3,6,20,26,35,38,41,43,44,49,60,67,70,81,84,89,97,109,119,122,123,129,130,132,134,141,144,152,158,159,165,171,174,179,182,186,187,196]

**Comment by stalin18** [comment]
@JiaRenChang : "[1,3,6,20,26,35,38,41,43,44,49,60,67,70,81,84,89,97,109,119,122,123,129,130,132,134,141,144,152,158,159,165,171,174,179,182,186,187,196]" -- Is this confirm, these are 39 images, I expected that it would be 40 (validation dataset has 20% images -- in your paper)

**Comment by JiaRenChang** [comment]
@stalin18 
Sorry for my mistake. The image 184 was also included in the validation set.

[1,3,6,20,26,35,38,41,43,44,49,60,67,70,81,84,89,97,109,119,122,123,129,130,132,134,141,144,152,158,159,165,171,174,179,182, **184**, 186,187,196]
Total 40 images for validation

**Comment by xianzhidu** [comment]
@JiaRenChang Do these numbers represent the filenames? Such as '1' corresponds to '000001_10.png'? I got a validation error rate 1.09% (1.98% in your paper) if I do it this way on KITTI2015.

**Comment by verigle** [comment]
image = [img for img in os.listdir(filepath+left_fold) if img.find('_10') > -1]

  train = image[:160]
  val   = image[160:]

you can print(val) to get real validation sets

**Comment by zhFuECL** [comment]
@xianzhidu  I have the same problem. Have you solved it?

**Comment by JiaRenChang** [comment]
@xianzhidu @zhFuECL 
You should get lower validation error rate because that the public model was trained 1000 epochs (for submission).

The reported validation error in the Table is the model trained 300 epochs (for ablation study).
